### PR TITLE
Add new role rules and additional roles page that starts with just web designer

### PR DESCRIPTION
**Pull request by gflofst**

### CODE REVIEW

**Review by JulianKunkel — APPROVED** [review]

**Review by seattleplus — DISMISSED** [review]

**Review by adilger — COMMENTED** [review]
As general advice - this change that references the new `foundation-new-roles` and `additional-roles` pages should be in a single commit with the addition of those pages, so that there is not a time when the website is "broken".  That is less of an issue with the website than an application which can't compile, but a good practice nonetheless.

File: templates/Pages/foundation-new-roles [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+<li>The Web Designer will not be allowed or asked to vote on any Committee issues.</li>
+
+<li>The Web Designer will not have access to the shared IO500 Google Drive, nor the meeting minutes since they include sensitive information.</li>
+
+<li>This Web Designer’s Slack access will be limited to a new private slack channel dedicated to the role and will not have access to the other private Committee channels except as deemed necessary based on future Committee unanimous consent. </li>
+
+<li>The role is perpetual unless the person resigns. At any time, if one Committee member has a concern with the position, s/he can discuss the matter in a timely fashion with the Committee, if the concerns of the Committee member cannot be resolved in the Committee discussion, the current Web Designer is removed immediately based on a majority vote. Prior to notifying the web designer, all passwords and password recovery methods must be changed/verified to remove the web designer’s access. This would include the webpage github, hosting, disqus, email lists, slack, and any other related functions.</li?
```
Comment (adilger) [inline templates/Pages/foundation-new-roles]: Better to use a neutral pronoun like "they" instead of s/he.

**Review by adilger — APPROVED** [review]
IMHO, it is better to have changes like this merged together *before* they are pushed to Github for landing.  That avoids reviewing one patch, only to find that the next patch is fixing a bunch of things in the first one, deleting text in a later patch, etc.  That doesn't apply if the changes are complex and more easily reviewed as a series of incremental changes, but _does_ apply to a series like this that has a bunch of fixes to patches earlier in the series.

To merge patches locally, you can use `git commit -a --amend` locally to merge all of your current changes into the previously committed patch.  If there are several patches already committed locally you can use `git rebase -i main` to invoke the interactive patch manager, and then `squash` the later fix commits into the earlier ones.

**Comment by gflofst** [comment]
Agreed on the single merge request. I just did the edits via the webpage rather than on my machine. I will endeavor to try to make it a single commit in the future.

**Review by adilger — APPROVED** [review]

**Comment by adilger** [comment]
Jay, note that you can use "[Rebase and merge]" when landing a patch series like this, so that it is merged into a single commit in the main repository.

**Review by JulianKunkel — APPROVED** [review]

**Comment by gflofst** [comment]
@gmarkomanolis @seattleplus, are these changes ok?

**Review by gmarkomanolis — APPROVED** [review]

**Review by JulianKunkel — APPROVED** [review]

**Comment by gflofst** [comment]
@seattleplus, can you approve this since it is a rules change?

**Comment by gflofst** [comment]
@seattleplus, can you approve this since it is a rules change?

**Review by seattleplus — APPROVED** [review]

**Pull request merged**